events.md

```diff
@@ -50,6 +50,15 @@ Argumenty:
 
 ---
 
+## `setPosition`
+
+Argumenty:
+  * `arg1`: ID ustawionej lokacji
+
+Event ten jest podnozszony gdy lokacja zostaje ustawiona (recznie lub automatycznie)
```

Comment:
moze warto byloby dodac argument ktory mowilby o zrodle ustawienia?

Reply:
W sensie, że np. "afterFailledFollow"?

Reply:
Hmm wiąże się to z dodanie parametru dla każdego z wywołań, skórka nie warta wyprawki chyba.